ci/scripts/collect-deb-package.sh

```diff
@@ -26,7 +26,13 @@ tar --exclude={".git","build"} -czpf centreon-collect-$VERSION.tar.gz "$ROOT"
 cd "$ROOT"
 cp -rf ci/debian-collect debian
 sed -i "s/^centreon:version=.*$/centreon:version=$(echo $VERSION-$RELEASE)/" debian/substvars
-#sed -i "s/^centreon:version=.*$/centreon:version=$(echo $VERSION | egrep -o '^[0-9][0-9].[0-9][0-9]')/" debian/substvars
-debmake -f "${AUTHOR}" -e "${AUTHOR_EMAIL}" -u "$VERSION" -r "$DISTRIB"
+echo "debmake begin"
+debmake -f "${AUTHOR}" -e "${AUTHOR_EMAIL}" -u "$VERSION" -r "$RELEASE"
```

Comment:
Wouldn't it be interesting to try -r "$RELEASE-$DISTRIB" or something like this to get exactly what you wanted initially?

Reply:
I wanted to do this too, but given how we are going to structure repositories, it does not make sense anymore 
Repos will be structured around distributions (bullseye, buster, etc) like yum is